### ISSUE TITLE
Can't find the theme option file.

### ISSUE THREAD

**Issue by Bruno21** [thumbsup/thumbsup#218]
Hello, 

First, thanks for sharing this great tool. I was looking for a tool to replace the Synology Photostation app for my NAS.
I successfully run thump from Docker, except I cannot load the theme config file.

Here my install install:
```
bruno@DS916:/volume1/photo/thumbsup $ l
drwxr-xr-x    1 bruno    users          222 Nov 15 13:28 .
drwxrwxrwx    1 PhotoSta PhotoSta       570 Nov 15 13:29 ..
-rwxrwxrwx    1 bruno    users          171 Nov 15 13:43 install2.sh
drwxr-xr-x    1 bruno    users          140 Nov 15 15:08 website
```
```
#/bin/bash -e
docker run \
  -v /volume1/photo/Flore:/input:ro \
  -v `pwd`/website:/output \
  thumbsupgallery/thumbsup:latest \
  thumbsup --config /input/config.json

```
Here my Photos source folder:
```
bruno@DS916:/volume1/photo/Flore $ l
drwxrwxrwx    1 bruno    PhotoSta      1706 Nov 15  2016 Arbres
drwxrwxrwx    1 bruno    PhotoSta       136 Mar  8  2016 Champignons
drwxrwxrwx    1 bruno    PhotoSta      2592 Nov 15  2016 Fleurs
-rwxrwxrwx    1 bruno    users          332 Nov 15 15:08 config.json
-rwxrwxrwx    1 bruno    users           24 Nov 15 12:51 theme_options.json
```
and my config.json:
```
{
        "input": "/input",
        "output": "/output",
        "thumb-size": 250,
        "large-size": 1600,
        "photo-download": "symlink",
        "cleanup": true,
        "sort-albums-by": "title",
        "sort-media-by": "date",
        "sort-media-direction": "desc",
        "home-album-name": "Accueil",
        "theme": "flow",
        "theme-settings: /input/theme_options.json",
        "title": "Photos...",
        "embed-exif": true,
        "locale": "fr"
}
```
I've tried many paths for "theme-settings: /input/theme_options.json" without luck.

If running as a Docker container:
```
Thumbsup image tag: latest
Docker version: Docker version 18.09.8, build bfed4f5
Operating system: Synology DSM 6.2
```

**Comment by rprieto** [comment]
Hi @Bruno21. You have the right syntax. I ran some tests and passing `"theme-settings: /input/theme_options.json"` works well for me. Which theme options were you trying to use?

I notice that when you created the issue the latest Docker image 1.14.0. The `flow` theme bundled in this version didn't have the Featured Album feature yet. I tried with version 1.15.0 and it works as expected using

```js
// /input/theme_options.json
{
  "featuredAlbum": "test"
}
```
